src/styles/snake/antisnake.scss

```diff
@@ -0,0 +1,42 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    color: #fff;
+    height: 300px;
+  }
+
+  &__cell--1 {
+    background-color: rgb(100%, 0%, 0%);
+  }
+  &__cell--2 {
+    background-color: rgb(80%, 0%, 0%);
+  }
+
+  &__cell--3 {
+    background-color: rgb(60%, 0%, 0%);
+  }
+  &__cell--4 {
+    background-color: rgb(40%, 0%, 0%);
+  }
+  &__cell--5 {
+    background-color: rgb(20%, 0%, 0%);
+  }
+  &__cell--6 {
+    background-color: rgb(0%, 0%, 0%);
+  }

```

Comment:
9-st point of the CHECKLIST 👉 [checklist.md](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
You can rewrite it by using @for or @each cycles, as well.